_includes/configuration/paste-merge-formats.md

```diff
@@ -0,0 +1,26 @@
+{% if page.title == "Paste plugin" %}
+  {% assign plugin = "paste" %}
+  {% assign pluginname = "Paste" %}
+{% else %}
+  {% assign plugin = "powerpaste" %}
+  {% assign pluginname = "PowerPaste" %}
+{% endif %}
+### `paste_merge_formats`
+
+This option enables the merge format feature of the {{pluginname}} plugin. This merges similar text formats to reduce the number of HTML elements produced. For example: `<b>abc <b>bold</b> 123</b>` becomes `<b>abc bold 123</b>` since the inner format is redundant. This option is enabled by default but can be disabled if retaining nesting or similar formats is important.
```

Comment:
Do we wanna clarify that the elements have to match on name, attrs, inline styles and classes (iirc)? Just wondering if "similar text formats" and the basic example might be... too simple an explanation?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1576/commits/ddffa8b980ada746418f251acd6de818dc4ab381